packages/dashboard/src/customization/widgets/lineScatterChart/component.tsx

```diff
@@ -200,8 +203,7 @@ const LineScatterChartWidgetComponent: React.FC<LineScatterChartWidget> = (widge
         thresholds={thresholds}
         significantDigits={significantDigits}
         size={size}
-        legend={legend?.visible == true ? {} : undefined}
-        defaultVisualizationType={mapConnectionStyleToVisualizationType(line?.connectionStyle)}
```

Comment:
why did you remove this defaultVisualizationType ?

Reply:
Good catch, accidentally removed when resolving merge conflicts.